README.md

```diff
@@ -1,2 +1,43 @@
 # Personal-Portfolio
+
 This is my Microverse Personal Portfolio.
+
+> Week one Day 3 task
+
+## Built With
+
+- HTML/CSS
+
+## Getting Started
+
+- Click on the clone button to copy the clone
+- Open Terminal and run git clone <copied address>
+- Clone the repo
+- From you terminal run " code ." to open in your code editor.
+
+**Setting up My Personal Portolio task**
+
+## Authors
+
+👤 **Promise Uka**
+
+- GitHub: [@githubhandle](https://github.com/Ukaypromise/)
+- Twitter: [@twitterhandle](https://twitter.com/PromiseUkay)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/engr-promise-uka-5a1450173/)
```

Comment:
[OPTIONAL CHANGES]
 Kindly add your handle like - Github: [@Ukaypromise ](https://github.com/Ukaypromise/) using `Github: [@Ukaypromise ](https://github.com/Ukaypromise/)` to customise your README.md file fully.

Reply:
Thank you so much.